tools/skstat.in

```diff
@@ -744,13 +744,14 @@ class BusManager:
 
         # attempt to get the qdrouterd process memory usage
         # this may not be present on all platforms
-        router = self.query('io.skupper.router.router')[0]
+        router_metrics = self.query('io.skupper.router.routerMetrics')[0]
```

Comment:
Darn ! It looks like I introduced this bug !! Thanks for fixing it

Reply:
np - the real bug was in the CI test I wrote that didn't verify these metrics were present.  Had I done that correctly we would've caught this in development.